forms.html

```diff
@@ -5,45 +5,108 @@
         <meta name="viewport" content="width=device-width, initial-scale=1.0">
         <title>Registration Form</title>
         <style>
-            body {font-family: Arial;}
-            legend {
-                background-color: rgb(188, 182, 182);
-                color: black;
-                padding: 5px 10px
-            }
-            fieldset {
-                background-color: #eeeeee;
-            }
+            body {
+                background-color: rgb(236, 236, 236);
+        }
+            form {
+                background-color: white;
+                padding: 25px;
+                margin-left: 36%;
```

Comment:
Instead of setting specific margin values you can use `margin: 0 auto` to center the element horizontally. 
You just need to set specific width to your element and center it.

Reply:
nice, it solved my problem with resizing the window :), thank you :)